app/src/main/java/lt/vilnius/tvarkau/views/adapters/ProblemImagesPagerAdapter.java

```diff
@@ -0,0 +1,53 @@
+package lt.vilnius.tvarkau.views.adapters;
+
+import android.content.Context;
+import android.support.v4.view.PagerAdapter;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import lt.vilnius.tvarkau.R;
+
+/**
+ * Created by Gediminas Zukas on 2016-04-28.
+ */
+public class ProblemImagesPagerAdapter extends PagerAdapter {
+
+    private LayoutInflater mLayoutInflater;
+    private int[] mResources;
+
+    public ProblemImagesPagerAdapter(Context context, int[] imagesIds) {
+        mLayoutInflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        mResources = imagesIds;
+    }
+
+    @Override
+    public int getCount() {
+        return mResources.length;
+    }
+
+    @Override
+    public boolean isViewFromObject(View view, Object object) {
+        return view == object;
+    }
+
+    @Override
+    public Object instantiateItem(ViewGroup container, int position) {
+        // TODO: consider recycling views
+        View itemView = mLayoutInflater.inflate(R.layout.problem_images_view_pager_item, container, false);
+
+        ImageView problemImageView = (ImageView) itemView.findViewById(R.id.problem_image_view);
+        problemImageView.setImageResource(mResources[position]);
+
+        container.addView(itemView);
+
+        return itemView;
```

Comment:
There is few messages in logcat while swipping images:
```
04-29 15:22:04.316 4339-4339/lt.vilnius.tvarkau I/Choreographer: Skipped 44 frames!  The application may be doing too much work on its main thread.
```
It should fix itself with future PR of loading images from urls. 

If you want to fix it know just load images with `Picaso`
```java
Picasso.with(container.getContext()).load(mResources[position]).into(problemImageView);
```